<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_site</class>
 <widget class="QDialog" name="Add_site">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Свойства участка</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Время участка, часы</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Угол дрейфа корабля, градусы</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Скорость ветра, м/сек</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TimeSite">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="BettaAngle">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="SpeedWind">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>361</width>
     <height>52</height>
    </rect>
   </property>
   <property name="title">
    <string>Направление ветра</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="WindBoard">
        <property name="text">
         <string> В борт</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="WindHead">
        <property name="text">
         <string>Встречный</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="WindFair">
        <property name="text">
         <string>Попутный</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="WindScula">
        <property name="text">
         <string>В скулу</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="AcceptedButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Ок</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CencelButton">
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
